[Snyk] Upgrade diff from 5.0.0 to 5.1.0

alikon opened the pull request:
<p>This PR was automatically created by Snyk using the credentials of a real user.</p><br /><h3>Snyk has created this PR to upgrade diff from 5.0.0 to 5.1.0.</h3>

:information_source: Keep your dependencies up-to-date. This makes it easier to fix existing vulnerabilities and to more quickly identify and fix newly disclosed vulnerabilities when they affect your project.
<hr/>

- The recommended version is **1 version** ahead of your current version.
- The recommended version was released **2 years ago**, on 2022-05-23.


<details>
<summary><b>Release notes</b></summary>
<br/>
  <details>
    <summary>Package name: <b>diff</b></summary>
    <ul>
      <li>
        <b>5.1.0</b> - <a href="https://snyk.io/redirect/github/kpdecker/jsdiff/releases/tag/v5.1.0">2022-05-23</a></br><p>… (<a class="issue-link js-issue-link" data-error-text="Failed to load title" data-id="1218995979" data-permission-text="Title is private" data-url="https://github.com/kpdecker/jsdiff/issues/365" data-hovercard-type="pull_request" data-hovercard-url="/kpdecker/jsdiff/pull/365/hovercard" href="https://snyk.io/redirect/github/kpdecker/jsdiff/pull/365">#365</a>)</p>
<p>Co-authored-by: Tony Spataro <a href="mailto:anthony.spataro@appfolio.com">anthony.spataro@appfolio.com</a></p>
      </li>
      <li>
        <b>5.0.0</b> - <a href="https://snyk.io/redirect/github/kpdecker/jsdiff/releases/tag/v5.0.0">2020-11-08</a></br><p>5.0.0</p>
      </li>
    </ul>
    from <a href="https://snyk.io/redirect/github/kpdecker/jsdiff/releases">diff GitHub release notes</a>
  </details>
</details>
<hr/>

**Note:** *You are seeing this because you or someone else with access to this repository has authorized Snyk to open upgrade PRs.*

For more information:  <img src="https://api.segment.io/v1/pixel/track?data=eyJ3cml0ZUtleSI6InJyWmxZcEdHY2RyTHZsb0lYd0dUcVg4WkFRTnNCOUEwIiwiYW5vbnltb3VzSWQiOiJjZDIwN2NhZS04MzA5LTQzYWMtOWVhMy0wZjFlNzRjZTdiZmYiLCJldmVudCI6IlBSIHZpZXdlZCIsInByb3BlcnRpZXMiOnsicHJJZCI6ImNkMjA3Y2FlLTgzMDktNDNhYy05ZWEzLTBmMWU3NGNlN2JmZiJ9fQ==" width="0" height="0"/>

🧐 [View latest project report](https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🛠 [Adjust upgrade PR settings](https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f/settings/integration?utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr)

🔕 [Ignore this dependency or unsubscribe from future upgrade PRs](https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f/settings/integration?pkg&#x3D;diff&amp;utm_source&#x3D;github&amp;utm_medium&#x3D;referral&amp;page&#x3D;upgrade-pr#auto-dep-upgrades)

<!--- (snyk:metadata:{"prId":"cd207cae-8309-43ac-9ea3-0f1e74ce7bff","prPublicId":"cd207cae-8309-43ac-9ea3-0f1e74ce7bff","dependencies":[{"name":"diff","from":"5.0.0","to":"5.1.0"}],"packageManager":"npm","type":"auto","projectUrl":"https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f?utm_source=github&utm_medium=referral&page=upgrade-pr","projectPublicId":"7f972e16-a9b0-41f9-a764-0801652e912f","env":"prod","prType":"upgrade","vulns":[],"issuesToFix":[],"upgrade":[],"upgradeInfo":{"versionsDiff":1,"publishedDate":"2022-05-23T02:22:24.851Z"},"templateVariants":[],"hasFixes":false,"isMajorUpgrade":false,"isBreakingChange":false,"priorityScoreList":[]}) --->

richard67 commented:
I've tested this already here on MySQL and PostgreSQL, both with success.

richard67 commented:
Off topic: I have checked [https://github.com/joomla/joomla-cms/pull/21851/files#diff-a85cd4073fd85d63a651557b2b3465aeR111](https://github.com/joomla/joomla-cms/pull/21851/files#diff-a85cd4073fd85d63a651557b2b3465aeR111). It looks ok to me. The first statement for a table is converting the character set of the table, the next statement for the same table a bit below is then changing the default character set, which determins which character set will be used for columns which will be added in future. So this line you marked is ok and not a duplicate of the change below.

richard67 commented:
Off topic: Is it a known problem that installing 4.0 does not work with PostgreSQL?

alikon commented:
i haven't  tryed recently postgresql on 4.0 cause of "workflow" storm.....
please open an issue ......

richard67 commented:
maybe tomorrow

snyk-bot committed:
fix: upgrade diff from 5.0.0 to 5.1.0

Snyk has created this PR to upgrade diff from 5.0.0 to 5.1.0.

See this package in npm:
https://www.npmjs.com/package/diff

See this project in Snyk:
https://app.snyk.io/org/alikon/project/7f972e16-a9b0-41f9-a764-0801652e912f?utm_source=github&utm_medium=referral&page=upgrade-pr
